packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -203,4 +214,41 @@ describe('Marketplace page', () => {
     expect(submitProviderText).toBeVisible();
     expect(pluginCardText).toEqual(null);
   });
+
+  it('only show install button for plugins already installed', async () => {
```

Comment:
Shouldn't this be `it shows the installed text for installed plugins`?

```suggestion
  it('only shows the install button for plugins already installed', async () => {
```